src/main/java/org/prebid/server/spring/config/SettingsConfiguration.java

```diff
@@ -214,6 +223,110 @@ public JdbcPeriodicRefreshService ampJdbcPeriodicRefreshService(
         }
     }
 
+    @Configuration
+    @ConditionalOnProperty(prefix = "settings.s3", name = {"accounts-dir", "stored-imps-dir", "stored-requests-dir"})
+    static class S3SettingsConfiguration {
+
+        @Component
+        @ConfigurationProperties(prefix = "settings.s3")
+        @ConditionalOnProperty(prefix = "settings.s3", name = {"accessKeyId", "secretAccessKey"})
+        @Validated
+        @Data
+        @NoArgsConstructor
+        private static class S3ConfigurationProperties {
+            @NotBlank
+            private String accessKeyId;
+            @NotBlank
+            private String secretAccessKey;
+            @NotBlank
+            private String endpoint;
+            @NotBlank
+            private String bucket;
+            @NotBlank
+            private String accountsDir;
+            @NotBlank
+            private String storedImpsDir;
+            @NotBlank
+            private String storedRequestsDir;
+            @NotBlank
+            private String storedResponsesDir;
+        }
+
+        @Bean
+        S3AsyncClient s3AsyncClient(S3ConfigurationProperties s3ConfigurationProperties) throws URISyntaxException {
+            final AwsBasicCredentials credentials = AwsBasicCredentials.create(
+                    s3ConfigurationProperties.getAccessKeyId(),
+                    s3ConfigurationProperties.getSecretAccessKey());
+
+            return S3AsyncClient
+                    .builder()
+                    .credentialsProvider(StaticCredentialsProvider.create(credentials))
+                    .endpointOverride(new URI(s3ConfigurationProperties.getEndpoint()))
+                    .region(Region.EU_CENTRAL_1)
```

Comment:
:warning:  this is something that needs either to be configurable or documented on how to set this with `env` variables

Reply:
https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/region-selection.html

Reply:
This has to be configurable.